core/src/main/scala/cromwell/core/path/PathFactory.scala

```diff
@@ -58,7 +58,7 @@ object PathFactory {
     path.map(postMapping) getOrElse {
       val pathBuilderNames: String = pathBuilders map { _.name } mkString ", "
       throw PathParsingException(
-        s"Either $string exists on a filesystem not supported by this instance of Cromwell, or a failure occurred while building an actionable path from it." +
+        s"""Either the path "$string" exists on a filesystem not supported by this instance of Cromwell, or a failure occurred while building an actionable path from it.""" +
```

Comment:
preexisting but what's an "actionable" path?

Reply:
I restrained myself from doing a comprehensive rewording, but now you're enabling me